platform/barefoot/bfn-platform.mk

```diff
@@ -1,5 +1,5 @@
-BFN_PLATFORM = bfnplatform_9.0.0.cc6ccbe_pr_deb9.deb
-$(BFN_PLATFORM)_URL = "https://github.com/barefootnetworks/sonic-release-pkgs/raw/rel_9_0/bfnplatform_9.0.0.cc6ccbe_pr_deb9.deb"
+BFN_PLATFORM = bfnplatform_1.0.0_amd64.deb
```

Comment:
we will correct this in next commits when server is ready.

Reply:
ok